src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -40,6 +40,9 @@
     <Compile Include="System\IO\Path.IsPathFullyQualified.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="System\BitConverter.netcoreapp.cs" />
+    <Compile Include="System\BitConverterArray.cs" />
+    <Compile Include="System\BitConverterBase.cs" />
```

Comment:
I split the test classes into three, with BitConverter.netcoreapp.cs as the original file and BitConverterArray.cs handling all byte array methods, BitConverterSpan.cs all span methods, and BitConverterBase as the base class for the last two.

Let me know if you have thoughts about naming conventions / organization here @stephentoub